ui/pages/swaps/dropdown-search-list/dropdown-search-list.js

```diff
@@ -81,13 +113,74 @@ export default function DropdownSearchList({
     }
   };
 
+  const ImportTokenModalFooter = (
+    <>
+      <Button
+        type="secondary"
+        className="page-container__footer-button"
+        onClick={onImportTokenCloseClick}
+        rounded
+      >
+        {t('cancel')}
+      </Button>
+      <Button
+        type="confirm"
+        className="page-container__footer-button"
+        onClick={onImportTokenClick}
+        rounded
+      >
+        {t('import')}
+      </Button>
+    </>
+  );
+
   return (
     <div
       className={classnames('dropdown-search-list', className)}
       onClick={onClickSelector}
       onKeyUp={onKeyUp}
       tabIndex="0"
     >
+      {tokenForImport && isImportTokenModalOpen && (
+        <Popover
```

Comment:
I will abstract this modal window for importing a token to its own component and use that one here.

Reply:
Done.